src/main/java/org/entando/kubernetes/client/hub/domain/BundleGroupVersionEntityDto.java

```diff
@@ -0,0 +1,38 @@
+package org.entando.kubernetes.client.hub.domain;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * This is the exact match of com.entando.hub.catalog.persistence.entity.BundleGroupVersion.
+ * TODO find usage in the FE and and possibly use the proper dto
+ */
```

Comment:
Please remove this JPA annotationplease remove this comment because inside the component manager it has no meaning (the CM does not know the db structure)

Reply:
Done